tests/test_series.py

```diff
@@ -681,8 +709,8 @@ def test_types_scalar_arithmetic() -> None:
     res_pow3: pd.Series = s.pow(0.5)
 
 
-# GH 103
 def test_types_complex_arithmetic() -> None:
+    # GH 103
     c = 1 + 1j
     s = pd.Series([1.0, 2.0, 3.0])
     x = s + c
```

Comment:
could convert these as well (with a TODO)

Reply:
Same as above, right now the typehinting would lead to `pd.Series[Unknown]` so would rather make a separate PR to fix the typehints altogether.

Reply:
OK, that makes sense